### PR TITLE
Added basic documentation generation infrastructure

### LINKED ISSUE #7: Initiate documentation building
- Making sure we embed generated reportlets within the explanations
- Integration of jupyter-books? (cc/ @effigies )

### PR DESCRIPTION
**Pull request by ZviBaratz**
Closes #7.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = "nireports"
+copyright = "2022, The NiPreps Developers"
+author = "The NiPreps Developers"
+
+# The full version, including alpha/beta/rc tags
+release = "0.1.0"
```
Comment (oesteban) [inline docs/conf.py:25]: Please add somewhere at the top:

```Python
try:
    # This option is only available with Python 3.8
    from importlib.metadata import version as get_version
except ImportError:
    from importlib_metadata import version as get_version
```

And then this line would go like:

```suggestion
release = get_version("nireports")
```

File: docs/installation.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Installation
+============
+
```
Comment (oesteban) [inline docs/installation.rst:3]:
```suggestion
```

File: docs/modules/nireports.reportlets.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+nireports.reportlets package
+============================
+
```
Comment (oesteban) [inline docs/modules/nireports.reportlets.rst:3]:
```suggestion
```

File: docs/modules/nireports.rst [hunk 4]
```diff
@@ -0,0 +1,18 @@
+nireports package
+=================
+
```
Comment (oesteban) [inline docs/modules/nireports.rst:3]:
```suggestion
```

File: docs/overview.rst [hunk 5]
```diff
@@ -0,0 +1,5 @@
+Overview
+========
+
```
Comment (oesteban) [inline docs/overview.rst:3]:
```suggestion
```

File: docs/quickstart.rst [hunk 6]
```diff
@@ -0,0 +1,4 @@
+Quickstart
+==========
+
```
Comment (oesteban) [inline docs/quickstart.rst:3]:
```suggestion
```

**Review by oesteban — APPROVED** [review]
Mostly stylistic changes. Just an addition regarding version retrieving.

**Review by eilidhmacnicol — APPROVED** [review]

**Pull request merged**